llama-index-integrations/llms/llama-index-llms-openai/llama_index/llms/openai/base.py

```diff
@@ -544,6 +553,85 @@ def _get_response_token_counts(self, raw_response: Any) -> dict:
             "total_tokens": usage.get("total_tokens", 0),
         }
 
+    def _get_tool_call(
+        self,
+        response: ChatResponse,
+    ) -> OpenAIToolCall:
+        tool_calls = response.message.additional_kwargs.get("tool_calls", [])
+
+        if len(tool_calls) < 1:
+            raise ValueError(
+                f"Expected at least one tool call, but got {len(tool_calls)} tool calls."
+            )
+
+        # TODO: support more than one tool call?
```

Comment:
out of curiosity what do we need to do to support parallel function calling here?